solr/core/src/java/org/apache/solr/core/SchemaCodecFactory.java

```diff
@@ -23,9 +23,9 @@
 import org.apache.lucene.codecs.DocValuesFormat;
 import org.apache.lucene.codecs.KnnVectorsFormat;
 import org.apache.lucene.codecs.PostingsFormat;
-import org.apache.lucene.codecs.lucene92.Lucene92Codec;
-import org.apache.lucene.codecs.lucene92.Lucene92Codec.Mode;
-import org.apache.lucene.codecs.lucene92.Lucene92HnswVectorsFormat;
```

Comment:
I'm not sure why this changed from 91 to 92 in the 9.3 update in #960 but presume it should change to 94 for 9.4 here if possible?

Reply:
The branch was called 9.2 because that's what I started on, but eventually moved on to 9.3 - there is nothing significant about the naming.

The codec is 92 because 93 didn't get new codecs.